src/Development/Shake/FileInfo.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving, DeriveDataTypeable, CPP, ForeignFunctionInterface #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving, DeriveDataTypeable, CPP #-}
 
```

Comment:
Isn't this wrong on Windows? Where I do FFI in the ifdef? I think this needs fixing before merging.